frontend/src/metabase/setup/components/NewsletterForm/NewsletterForm.styled.tsx

```diff
@@ -26,7 +26,7 @@ export const EmailFormLabelCard = styled.div`
   display: flex;
   padding: 0 1.5rem;
   color: ${color("text-medium")};
-  background-color: ${color("white")};
+  background-color: var(--mb-color-bg-white);
```

Comment:
I've noticed a few times we convert from a `color()` call to a `var` css class, and the other way around as well. We also have the Mantine theme calls for colors as well. What should we try to default to in the future?